relational_operators/SampleOperator.cpp

```diff
@@ -0,0 +1,121 @@
+/**
+ *   Copyright 2016, Quickstep Research Group, Computer Sciences Department,
+ *   University of Wisconsin—Madison.
```

Comment:
This was copy pasted from another file. Most of the copy right text in the other files follows the same formatting